spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/web/SpringDataWebProperties.java

```diff
@@ -59,6 +59,28 @@ public Sort getSort() {
 		 */
 		private int defaultPageSize = 20;
 
+		/**
+		 * Configures a general prefix to be prepended to the page number and page size parameters.
```

Comment:
Why a "configures a" all the sudden. Please look at existing descriptions for guidance

Reply:
Removed that now.